pkg/csi/service/node.go

```diff
@@ -789,13 +812,13 @@ func (s *service) NodeExpandVolume(
 	// NOTE(xyang): Make sure new size is greater than or equal to the
 	// requested size. It is possible for volume size to be rounded up
 	// and therefore bigger than the requested size.
-	if gotBlockSizeBytes < volSizeBytes {
-		return nil, status.Errorf(codes.Internal, "requested volume size was %d, but got volume with size %d", volSizeBytes, gotBlockSizeBytes)
+	if gotBlockSizeBytes < reqVolSizeBytes {
```

Comment:
gotBlockSizeBytes -> currentBlockSizeBytes

Reply:
Changed it.